### ISSUE TITLE
how to bundle Roboto web font?

### ISSUE THREAD

**Issue by cordovapolymer** [Polymer/polymer-bundler#453]
How to instruct polymer-bundler(I'm using it via polymer build) to bundle [Roboto](https://github.com/Polymer/font-roboto) and other external resources?
In polymer starter kit 1.0 gulp copy task has been doing this.

**Comment by justinfagnani** [comment]
@cordovapolymer are you using the font-roboto you linked to, which is very old and only compatible with Polymer 0.5, or the current one here: https://github.com/PolymerElements/font-roboto ?

@usergenic does this sound like we need an option to include certain external resources?

**Comment by web-padawan** [comment]
As i had already noticed in the other issue,  `font-roboto` can be easily overridden by `font-roboto-local` [like this](https://github.com/web-padawan/polymer-sandbox/blob/master/bower.json#L7) without rewriting any HTML imports. I also recommend using `gulp-ttf2woff` and `gulp-replace` to serve Roboro donts in `.woff` format.

**Comment by cordovapolymer** [comment]
@justinfagnani , you are right I've linked to a wrong font-roboto, I'm using the new one

**Comment by justinfagnani** [comment]
Yeah, I know about font-roboto-local, but that's a fairly one-off solution, and not really ideal for most web projects anyway, who probably should be using Google's CDN for cacheability. I think it'd be great if we had something that worked with external resources in general.

btw, @web-padawan are you building an Electron app or similar? That's the main use case for font-roboto-local.

**Comment by web-padawan** [comment]
@justinfagnani no, just a plain SPA with app-shell architecture served via `nginx` with no backend (only API requests). But we don't rely on service worker for now (because of Safari) and use fingerprinting assets via `gulp-rev-all` instead. So we decided to ship fonts from our repo as well (so they get fingerprinted once and should be cached permanently).

**Comment by cordovapolymer** [comment]
@justinfagnani , the are other use cases, we are building a cordova based app.
